### PR TITLE
Add coq-test

### PR DESCRIPTION
**Pull request by liyishuai**

### CODE REVIEW

File: extra-dev/packages/coq-test/coq-test.dev/opam [hunk 1]
```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+name: "coq-test"
+version: "dev"
```
Comment (clarus) [inline extra-dev/packages/coq-test/coq-test.dev/opam]: This is redundant with the package file name. I would remove these to avoid future inconsistencies.

**Comment by gares** [comment]
The description is a bit skinny, and the logPath a bit too generic. Would you mind writing a description and, if possible, choosing a logPath other than `Test`?

**Comment by liyishuai** [comment]
This library intends to be the basis of [QuickChick 2](https://github.com/QuickChick/QuickChick/issues/169). It aims to be generic for implementation and verification of testing in Coq. I don't have a good renaming in mind, but am open to suggestions. I can also suspend this release before QuickChick 2 makes more progress.

File: extra-dev/packages/coq-test/coq-test.dev/opam (hunk 1 above)
Comment (clarus) [reply extra-dev/packages/coq-test/coq-test.dev/opam]: Yes would be cool, thinking about it also, did not have time to make it yet.

**Comment by Lysxia** [comment]
I would suggest coq-testable-properties

**Comment by liyishuai** [comment]
Or `coq-testable` for short.
This library is quite experimental. Is there a guideline at which stage is a package good for extra-dev?

**Comment by gares** [comment]
I think extra Dev is mainly to let you use the package in ci. So no strong requirements.

I think names should be precise, it is not the case that you type them over and over.

**Comment by ybertot** [comment]
I concur with Enrico, names that are used rarely can be long.  coq-testable is not very explicit.  You said it was a basis for quickchick.  Why not coq-quickchick-testable?

**Comment by liyishuai** [comment]
Thanks everyone for the suggestions!
I'll reopen after the library is more developed than a playground.

**Pull request closed without merging**